projects/valtimo/dossier-management/src/lib/components/dossier-management-detail-container/dossier-management-detail-container.component.scss

```diff
@@ -14,6 +14,28 @@
  * limitations under the License.
  */
 
+::ng-deep.dossier-management-tabs .cds--tab-content {
+  background-color: var(--cds-layer);
+
+  &:focus {
+    outline: none;
+  }
+}
+
+::ng-deep.dossier-management-tabs .no-padding-left-right .cds--tab-content {
+  padding-left: 0;
+  padding-right: 0;
+}
+
+::ng-deep.dossier-management-tabs .no-padding-top-bottom .cds--tab-content {
+  padding-top: 0;
+  padding-bottom: 0;
+}
+
+::ng-deep.dossier-management-tabs .no-padding-bottom .cds--tab-content {
+  padding-bottom: 0;
+}
+
```

Comment:
All these can be grouped under one singular ::ng-deep.dossier-management-tabs .cds--tab-content